src/components/searchForm/searchForm.tsx

```diff
@@ -0,0 +1,19 @@
+import './searchForm.css';
+import { useState } from 'react';
+const SearchForm = ({initialSearchQuery, onSearch}: SearchFormProps) => {
+    const [searchText, setSearchText] = useState('');
```

Comment:
The question: is this component controlled or uncontrolled? Please think about the logic : this form is a leaf component. It receives with props everything it needs to work. And updates the search value via the passed callback (the state in the parent component)

Reply:
could you please provide me with more details ?
I'm not sure if i got your point